src/ui/public/vis/editors/config/editor_config_providers.ts

```diff
@@ -54,30 +54,73 @@ class EditorConfigProviderRegistry {
     return this.mergeConfigs(configs);
   }
 
+  private isBaseParam(config: EditorParamConfig): config is NumericIntervalParam {
+    return config.hasOwnProperty('base');
+  }
+
+  private isFixedParam(config: EditorParamConfig): config is FixedParam {
+    return config.hasOwnProperty('fixedValue');
+  }
+
+  private mergeHidden(current: EditorParamConfig, merged: EditorParamConfig): boolean {
+    return Boolean(current.hidden || merged.hidden);
+  }
+
+  private mergeFixedAndBase(
+    current: EditorParamConfig,
+    merged: EditorParamConfig,
+    paramName: string
+  ): { fixedValue?: any; base?: number } {
+    if (
+      this.isFixedParam(current) &&
+      this.isFixedParam(merged) &&
+      current.fixedValue !== merged.fixedValue
+    ) {
+      // In case multiple configurations provided a fixedValue, these must all be the same.
+      // If not we'll throw an error.
+      throw new Error(`Two EditorConfigProviders provided different fixed values for field ${paramName}:
+          ${merged.fixedValue} !== ${current.fixedValue}`);
+    } else if (
```

Comment:
I think this can be just a regular `if` instead of `else if`, since the first condition is an early exit. This will make the code a bit easier to scan.